client.go

```diff
@@ -1534,6 +1534,11 @@ func (c *Client) Exec(name string, cmd []string, env map[string]string,
 		"wait-for-websocket": true,
 		"interactive":        controlHandler != nil,
 		"environment":        env,
+		"forward-signals":    false,
+	}
+
+	if forwardSignal {
+		body["forward-signals"] = true
```

Comment:
Actually we don't I only put this there so that users can grep whether signals are forwarded or not. But I'll just push another commit on top that removes it.